chart/kubeapps/templates/apprepository-crd.yaml

```diff
@@ -1,7 +1,10 @@
+{{- if not (.Capabilities.APIVersions.Has "kubeapps.com/v1alpha1") -}}
```

Comment:
Would you mind adding a comment mentioning that this might have happened if another instance of kubeapps has been installed.